frontend_app_file/test/fixture/content/content.js

```diff
@@ -0,0 +1,12 @@
+import { file } from './file.js'
+import { commentList, revisionList } from 'tracim_frontend_lib/dist/tracim_frontend_lib.testUtils.js'
```

Comment:
you must not import from frontend_lib that way.
It must stay `tracim_frontend_lib`

Reply:
after some research, don't do this. It can't work